### PR TITLE
ipfs-cluster-ctl cmd: add no-status flag to pin add/rm

### PR DESCRIPTION
**Pull request by lanzafame**
@hsanjuan @ZenGround0 Thoughts on the naming of the shortflag `-nostat`?

### CODE REVIEW

File: ipfs-cluster-ctl/main.go [hunk 1]
```diff
@@ -305,7 +311,12 @@ in the cluster. The CID should disappear from the list offered by "pin ls",
 although unpinning operations in the cluster may take longer or fail.
 `,
 					ArgsUsage: "<CID>",
-					Flags:     []cli.Flag{},
+					Flags: []cli.Flag{
+						cli.BoolFlag{
+							Name:  "no-status, nostat",
+							Usage: "Prevents status reporting for faster unpinning",
```
Comment (hsanjuan) [inline ipfs-cluster-ctl/main.go]: same

File: ipfs-cluster-ctl/main.go [hunk 2]
```diff
@@ -305,7 +311,12 @@ in the cluster. The CID should disappear from the list offered by "pin ls",
 although unpinning operations in the cluster may take longer or fail.
 `,
 					ArgsUsage: "<CID>",
-					Flags:     []cli.Flag{},
+					Flags: []cli.Flag{
+						cli.BoolFlag{
+							Name:  "no-status, nostat",
+							Usage: "Prevents fetching pin status after pinning (faster, quieter)",
```
Comment (hsanjuan) [inline ipfs-cluster-ctl/main.go]: `unpinning`

**Comment by ZenGround0** [comment]
My personal preference is to omit a shortflag for this flag as `no-status` and `nostat` are close in pronunciation and length that having both feels a bit redundant to me.  I don't feel too strongly about this though so feel free to keep it as is if you disagree.

**Comment by lanzafame** [comment]
@ZenGround0 I don't disagree, though considering the purpose of this is to cut 1s from a pin/unpin operation, it would make sense to me to have a very short 'short' flag, i.e. `-ns`. @hsanjuan Thoughts?

**Comment by ZenGround0** [comment]
`-ns` SGTM

**Comment by hsanjuan** [comment]
yeah I prefer `-ns` too

**Comment by lanzafame** [comment]
Cool, I have just pushed the `-ns` short flag change up.

**Comment by hsanjuan** [comment]
Fixes #329

**Pull request merged**